### ISSUE TITLE
Incorrect ID when Importing Groups

### ISSUE THREAD

**Issue by rawkode** [terraform-providers/terraform-provider-gitlab#35]
### Terraform Version
v0.10.8

### Affected Resource(s)
- gitlab_group
- gitlab_project

### Expected Behavior
When importing a project or group, the ID should be set to the ID provided by GitLab

### Actual Behavior
The ID is set to the path from `statePassThrough`

### Steps to Reproduce
1. Create a GitLab group
2. Add resource to `.tf`
3. Import

### Important Factoids
This renders importing a group and then using it has a sub-group as useless, as it fails to convert the alphanumeric "id" to an integer.

**Comment by rawkode** [comment]
Hi @radeksimko, what would be the correct way to fix this? I have a working copy that does as I expect; but it doesn't feel like the correct solution:

```
commit c5ac978854cc559a3dc89f4b7e117fd7f0b875c6 (HEAD -> bugfix/import-id)
Author: David McKay <david.andrew.mckay@gmail.com>
Date:   Fri Oct 20 14:54:23 2017 +0100

    Ensuring that the IDs are set properly

diff --git a/gitlab/resource_gitlab_group.go b/gitlab/resource_gitlab_group.go
index 5097bca..572cebf 100644
--- a/gitlab/resource_gitlab_group.go
+++ b/gitlab/resource_gitlab_group.go
@@ -109,6 +109,7 @@ func resourceGitlabGroupRead(d *schema.ResourceData, meta interface{}) error {
                return err
        }
 
+       d.SetId(fmt.Sprintf("%d", group.ID))
        d.Set("name", group.Name)
        d.Set("path", group.Path)
        d.Set("description", group.Description)
diff --git a/gitlab/resource_gitlab_project.go b/gitlab/resource_gitlab_project.go
index 1581ebf..ab56c6f 100644
--- a/gitlab/resource_gitlab_project.go
+++ b/gitlab/resource_gitlab_project.go
@@ -94,6 +94,7 @@ func resourceGitlabProject() *schema.Resource {
 }
 
 func resourceGitlabProjectSetToState(d *schema.ResourceData, project *gitlab.Project) {
+       d.SetId(fmt.Sprintf("%d", project.ID))
        d.Set("name", project.Name)
        d.Set("path", project.Path)
        d.Set("description", project.Description)

```

**Comment by richardc** [comment]
That looks like the correct fix to me; because otherwise the state isn't going to be updated with the loaded id the underlying resource.

**Comment by richardc** [comment]
@rawkode I've submitted your fixes as #37 and #38 with some additional documentation to advertise how they now work.  Thanks for the report and time in figuring out the correct fix.
